ci-operator/config/openshift/ptp-operator/openshift-ptp-operator-release-4.18.yaml

```diff
@@ -36,6 +36,9 @@ promotion:
   - disabled: true
     name: "4.18"
     namespace: ocp
+  - disabled: true
```

Comment:
same here

Reply:
because the master/main configuration is pointing to 4.18 so we can only promote from one place at a time so have to disable it in the 4.18 configuration to handle that.